packages/esm-patient-test-results-app/src/index.ts

```diff
@@ -35,6 +29,17 @@ function setupOpenMRS() {
           columnSpan: 4,
         },
       },
+      {
+        id: "test-results-summary-dashboard",
+        slot: "patient-chart-dashboard-slot",
+        load: getAsyncLifecycle((() => {}) as any, options),
```

Comment:
I think this could be used to display the link in the nav bar.

Reply:
But does the dashboard MF need a _custom_ way of defining a link in the nav bar, do we have a use-case where we wouldn't use the default?

I don't like leaving the load function empty either. I thought we could set `meta.config.type` to `custom`, in which case the component would be displayed instead of a `GridView` or `TabbedView`

Reply:
Well, already the localization part is kind of magic right now and only works via meta and magic. Plus we don't know - some dashboards may want to display dynamic information (e.g., a badge) there.

So yes I'd say that would be the way forward.

Maybe the link could still come from the outside, and only the link's contents would be defined by the component.

Reply:
Alright, sounds good!

We don't have a way for an ExtensionSlot to define a fallback, in case a Extension does not provide a component, right? So even "default" behavior would be set explicitly in this case.

Reply:
With default behavior I mean: display a link with the title from the meta object.

Reply:
Yes correct!